src/Components/article-content.jsx

```diff
@@ -0,0 +1,53 @@
+import { getArticleById } from "../../api"
+import { useEffect, useState } from "react"
+import { useNavigate } from 'react-router-dom'
+
+
+function ArticleContent(props){
+    
+    const articleId = props.articleId
+    const [article, setArticle] = useState(null)
+    const navigate = useNavigate()
+
+    useEffect(()=>{
+        getArticleById(articleId)
+        .then((response)=>{
+            console.log(response)
```

Comment:
Remember to remove console logs by the time you put in pull requests - keep it clean